lib/ansible/modules/notification/mail.py

```diff
@@ -115,6 +116,17 @@
     - Sets the timeout in seconds for connection attempts.
     default: 20
     version_added: '2.3'
+  gnupg:
+    choices: [ true, false ]
+    description:
+      - Use OpenPGP encryption for the mail; incompatible with I(attach).
+      - Defaults to C(false), unless I(gnupg_recipients) is set.
+    version_added: '2.5'
+  gnupg_recipients:
+    default: []
+    description:
+      - A list of OpenPGP key fingerprints to encrypt the mail to.
```

Comment:
The description do not tell the format that should be used, could a example be added in the doc ?

Reply:
And as gundalow said on irc, explain a bit more the difference with recipient